### DISCUSSION TITLE
Bug Report - Sub editor lacks a feature from a changelog list

### DISCUSSION THREAD

**Discussion by seriousgranpa** [FakeFishGames/Barotrauma#14193] [Bug reports]
### Disclaimers

- [x] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

> - Instead of having to define the pirate sub files in the mission, you can also just leave it out and let the mission choose a random enemy sub. The reward, preferred difficulty, and tags that restrict which mission a sub can be used in, can be configured in the sub editor.

no, such options for enemy subs in an editor
![изображение](https://github.com/FakeFishGames/Barotrauma/assets/73081527/dab840cb-f631-44c9-baf8-8ff6b6285dfd)

### Reproduction steps

1. open game
2. open sub editor
3. create/load a sub
4. pick enemy type
5. no options for restricting missions

### Bug prevalence

Happens regularly

### Single player or multiplayer?

Single player

### Version

v1.5.8.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! It seems this change didn't actually make it into this update and was erroneously listed in the changelog. Should be included in the next update.
